### PR TITLE
Leon - Create ERD diagram and SQL script.

### PR DESCRIPTION
**Pull request by Leon-JS02**
Created an ERD diagram for the database.
Written an SQL script to initialise the database.

### CODE REVIEW

File: long_term/schema.sql [hunk 1]
```diff
@@ -1 +1,54 @@
...
+DROP TABLE IF EXISTS plant;
+DROP TABLE IF EXISTS origin_location;
+DROP TABLE IF EXISTS botanist;
+
+CREATE TABLE botanist(
+    botanist_id SMALLINT GENERATED ALWAYS AS IDENTITY,
+    first_name VARCHAR(255) NOT NULL,
+    last_name VARCHAR(255) NOT NULL,
+    email VARCHAR(255) NOT NULL UNIQUE,
+    phone VARCHAR(30) NOT NULL,
```
Comment (YakMan101) [inline long_term/schema.sql]: Add unique constraint. I don't think two botanists should have same number+extension

**Review by YakMan101 — CHANGES_REQUESTED** [review]
Being nitpicky:
- I think plant table plant_scientific_name should have unique constraint
- Might be worth making a unique constraint for origin_location where the combination of all (latitude, longitude, town, timezone) is unique.

**Review by YakMan101 — APPROVED** [review]

**Review by YakMan101 — APPROVED** [review]
Solid schema.

**Review by olliejth — APPROVED** [review]
Structure and constraints all LGTM.

**Pull request merged**